Dockerfile

```diff
@@ -1,14 +1,17 @@
-ARG DOCKER_BASE_IMAGE=klaytn/build_base:1.1-go1.15.7-solc0.4.24
+FROM golang:1.18-buster as builder
 
-FROM ${DOCKER_BASE_IMAGE}
-
-ENV PKG_DIR /locust-docker-pkg
 ENV SRC_DIR /go/src/github.com/klaytn/klaytn-load-tester
 ENV GOPATH /go
 
-RUN mkdir -p $PKG_DIR/bin
+WORKDIR $SRC_DIR
+ADD . .
+
+RUN (cd klayslave && \
```

Comment:
Hi @9to6, we have `make build` to build the klayslave binary. Can't we use make command in docker? Sorry, i'm not familar with docker so if this question makes no sense, please tell me.

Reply:
@yoomee1313 I've updated it.